styles.css

```diff
@@ -584,3 +595,329 @@ main {
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .main-margin {
+    margin-left: 18vw;
+  }
+
+  .about--container {
+    width: 64vw;
+    height: 122px;
+    display: flex;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .about--container h1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 60px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+  }
+
+  .description--container p {
+    width: 60vw;
+    height: 98px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .supporting--text {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+  }
+
+  #portfolio {
+    width: 96vw;
+    justify-content: center;
+  }
+
+  .description--container {
+    width: 740px;
+    height: 98px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .social--icons {
+    width: 740px;
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .portfolio {
+    width: 85vw;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    height: 496px;
+  }
+
+  .snapshot1,
+  .snapshot2,
+  .snapshot3,
+  .snapshot4 {
+    width: 40vw;
+    height: 448px;
+    background-repeat: no-repeat;
+    background-size: 100%;
+    margin-right: 22px;
+    margin-top: 8px;
+    border-radius: 24px;
+  }
+
+  .snapshot1 {
+    background-image: url("./images/desk1.svg");
+  }
+
+  .snapshot2 {
+    background-image: url("./images/portfolio3.png");
+  }
+
+  .snapshot3 {
+    background-image: url("./images/Snapshoot\ Portfolio.png");
+  }
+
+  .snapshot4 {
+    background-image: url("./images/Snapshoot_Portfolio2.png");
+  }
+
+  .primary-text {
+    width: 25vw;
+    height: 50px;
+    font-weight: 400;
+    font-size: 16px;
+  }
+
+  .primary-text p {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .portfolio-content {
+    width: 34rem;
+    height: 440px;
+    display: flex;
+    flex-direction: column;
+    padding-top: 24px;
+    gap: 14px;
+  }
+
+  .reverse {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    padding-left: 48px;
+  }
+
+  .title-heading {
+    width: 295px;
+    height: 36px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .container {
+    width: 95vw;
+    height: 846px;
+    background-image: url("./images/desk_footer-bg.svg");
+    background-size: 100%;
+    justify-content: center;
+    align-items: center;
+  }
+
```

Comment:
[OPTIONAL] I wouls suggest we increase the` border-top-left radius` to something like `130px `so it matches our figma design your design 👇 

![image](https://user-images.githubusercontent.com/97521557/201493828-45c61cf4-e038-4194-afbf-097f92fb3411.png)

Figma Design 👇 

![image](https://user-images.githubusercontent.com/97521557/201493846-a700bbac-5ce3-4210-b9be-6a90d99d4e09.png)

Reply:
Thank you @PinkMoon25 I will work on the suggested changes.